api/aws/dynamodb/v1/dynamodb.proto

```diff
@@ -35,6 +35,18 @@ service DDBAPI {
   }
 }
 
+// https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TableDescription.html
+enum Status {
+  UNSPECIFIED = 0;
```

Comment:
you'll want `UNKNOWN` as well if amazon returns a new status that you aren't able to convert into the enum